typings/jsdoc.namespace.js

```diff
@@ -14,7 +14,15 @@ module.exports = {
       }
     },
     newDoclet: ({ doclet }) => {
-      if (doclet.undocumented || doclet.memberof || !kinds.includes(doclet.kind)) return;
+      if (
+        doclet.undocumented ||
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['||' should be placed at the beginning of the line. (operator-linebreak)](https://app.codacy.com/app/DavertMik/CodeceptJS/pullRequest?prid=4500224)